classes/class3/cat_homework/app.js

```diff
@@ -0,0 +1,32 @@
+var express = require('express');
+var app = express();
```

Comment:
You should run this after all the requires. Good practice to include all dependencies before running any functionality.